CMakeLists.txt

```diff
@@ -19,6 +19,11 @@ if (UNIX)
    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -std=c++11")
 endif (UNIX)
 
+if (APPLE)
+   set(CMAKE_CXX_STANDARD 14)
```

Comment:
I think this CXX_STANDARD setting would make sense for all platform. Would you mind putting it outside the if?

Reply:
Sounds good! Will do!